code/modules/medical/genetics/combo_recipes.dm

```diff
@@ -133,7 +133,7 @@
 	result = /datum/bioEffect/fire_aura
 
 /datum/geneticsrecipe/strong
-	required_effects = list("fat","detox")
+	required_effects = list("dead_scan","detox")
```

Comment:
although this one feels a little weird. maybe the 'physically (un)fit' mutation instead of deadscan?

Reply:
I wasn't even aware that one existed :V 
My reasoning for it is that deadscan makes you show up as dead on a medscan while strong makes you show up as "very active" on a medscan. But yeah, physically unfit would make more sense

Reply:
Done!